src/content/api/module-methods.md

```diff
@@ -58,12 +59,13 @@ export default {
 
 ### `import()`
 
-`import('path/to/module') -> Promise`
+`function(String path):Promise`
```

Comment:
Let's stick with `string` https://github.com/webpack/webpack.js.org/search?q=string&unscoped_q=string

Reply:
my god where did i grab that Camel case :)

Reply:
updated